Python/other/Palindrome.py

```diff
@@ -0,0 +1,34 @@
+/*
+Python Program to check whether a number is Palindrome or not
+An integer is a palindrome if the reverse of that number is equal to the original number.
+Example of Palindromes are 11,44,101,121.
+*/
```

Comment:
Python does not use this. You can't be more serious using them in Python Code

Reply:
Sorry for the inconvenience. I will correct it